Arrowgene.Ddon.Server/GameLogicSetting.cs

```diff
@@ -98,6 +115,16 @@ public GameLogicSetting()
                 (7, 8, 0.6),
                 (9, 10, 0.5),
             };
+
+            GameClockTimescale = 90;
+
+            WeatherSequenceLength = 20;
+            WeatherStatistics = new List<(uint meanLength, uint weight)>
```

Comment:
Capitalize the first letter of `MeanLength` and `Weight` since these are used like class member variables.